xbmc/filesystem/VideoDatabaseDirectory/test/TestNodeMovieAssetTypes.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "filesystem/VideoDatabaseDirectory/DirectoryNode.h"
+#include "filesystem/VideoDatabaseDirectory/QueryParams.h"
+#include "test/TestUtils.h"
+#include "video/VideoDatabase.h"
+
+#include <gtest/gtest.h>
+
+using namespace XFILE::VIDEODATABASEDIRECTORY;
+
+TEST(TestNodeMovieAssetTypes, General)
+{
+  std::unique_ptr<CDirectoryNode> pNode(
```

Comment:
Remove the type prefix from the variable's name, please. Here, and everywhere else in this PR where you introduce new variables.